manual.adoc

```diff
@@ -211,5 +211,112 @@ In the example above, we've created two registries:
 
 Both registry have a `[shout]` block extension registered with a specific implementation.
 
-The first block extension is registered in the `registryA` and will transform the text to uppercase. 
+The first block extension is registered in the `registryA` and will transform the text to uppercase.
 The other one is registered in the `registryB` and will replace `.` by `{nbsp}!!!`.
+
+=== How to compile a Ruby extension to JavaScript (Node.js)
+
+Let's say you want to use the following Asciidoctor extension (written in Ruby) in your JavaScript based application.
+
+.lib/tel-inline-macro.rb
+```ruby
+require 'asciidoctor/extensions' unless RUBY_ENGINE == 'opal' # <1>
+
+include ::Asciidoctor
```

Comment:
For demo purposes, I don't think we should use this include. Let's just refer to all classes using their absolute name.

(I'm considering taking this out of the extensions lab too).

Reply:
I agree.
Is there any difference/advantage using `::Asciidoctor` (top level constant) ?